CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+0.37.1 / 2018-03-26
```

Comment:
I'd go with a minor version increment when changing dependencies and is this going to be released today or just at some point in the future?